src/flexmock/api.py

```diff
@@ -1037,11 +1069,8 @@ def _verify_call_order(cls, expectation: Expectation, args: Dict[str, Any]) -> N
             cls.last = next_method
         if expectation is not next_method and next_method is not None:
             raise CallOrderError(
-                "%s called before %s"
-                % (
-                    _format_args(str(expectation.name), args),
-                    _format_args(str(next_method.name), next_method._args),
-                )
+                f"{_format_args(str(expectation.name), args)} called before "
+                f"{_format_args(str(next_method.name), next_method._args)}"
```

Comment:
I'm thinking, should `format_args` just `str`ify whatever the first argument is ?

Reply:
I think it would be better to keep `_format_args` as it is because it doesn't make sense to receive anything else than a string. We should probably change Expecation somehow so that `name` can't be `None`. Now it seems that we create expectation without `name` when we create a partial mock.

Reply:
Yeah, that does not sound correct, let's fix it